services/scan/src/store.ts

```diff
@@ -167,10 +171,11 @@ export class Store {
   }
 
   /**
-   * Resets the database.
+   * Resets the database and any cached data in the store.
    */
   reset(): void {
     this.client.reset();
+    this.cachedLayouts = [];
```

Comment:
Are there any other times we need to invalidate the cache? E.g. when a new election is configured?

Reply:
When are the HMPB templates cleared from the database? Only when this `reset` function is called, which happens on unconfigure. So currently this is the only place we need to invalidate the cache. 

We only configure a new election after unconfigure. Separate from this caching question, maybe we should error out if we try to set the election without unconfiguring first.

Reply:
Ok great, that was my guess that this was the only place, but wanted to make sure. I do like the idea of putting in more guardrails somehow so that we make sure we don't mess up and forget to invalidate the cache if we change this down the road.